roles/matrix-dimension/templates/systemd/matrix-dimension.service.j2

```diff
@@ -0,0 +1,23 @@
+[Unit]
+Description=Matrix Dimension
+After=docker.service
+Requires=docker.service
+
+[Service]
+Type=simple
+ExecStartPre=-/usr/bin/docker kill matrix-dimension
+ExecStartPre=-/usr/bin/docker rm matrix-dimension
+ExecStart=/usr/bin/docker run --rm --name matrix-dimension \
+			--log-driver=none \
+			--cap-drop=ALL \
```

Comment:
Dropping capabilities is good, but we'd better also not start the container as `root`, but as a regular user.
It's done by passing a `--user` flag to the `docker run` command.

For other services we use the `matrix` user (e.g. `--user={{ matrix_user_uid }}:{{ matrix_user_gid }}`).

For Dimension, however, it seems like that won't work, as most other things are using the container's `node` user (uid=1000, gid=1000).

The Docker image can probably be improved to not be so picky, but for now, we can try to accommodate it.

I propose adding these additional variables to `roles/matrix-dimension/defaults/main.yml`:

```yaml
# The user and group id correspond to the node user in the `turt2live/matrix-dimension` image.
matrix_dimension_user_uid: 1000
matrix_dimension_user_gid: 1000
```

You can then adjust `matrix-dimension.service.j2` to contain a line like this:

```
			--user={{ matrix_dimension_user_uid }}:{{ matrix_dimension_user_gid }} \
```

For the image to be able to start though, it would need to access its `/data` files (at least `/data/config.yaml`).
We need to adjust permission for this too (see my other review comments for this).